magento/pwa-studio
# Applied Gift Cards show current balance

## Description

This PR adds current applied gift card balances to the UI to improve the UX.

The balance font size is slightly smaller than the code, so I consulted the Venia Styleguide for its value.

## Related Issue





Closes PWA-364.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the `/cart` page
1. Apply a gift card or two
1. See that the card's balance appears along with its code

## Screenshots / Screen Captures (if appropriate)

![Screen Shot 2020-02-10 at 12 22 20 PM](https://user-images.githubusercontent.com/13182778/74177871-30c2b880-4c00-11ea-8510-a3466026bfe6.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
